java/SQSSink/src/main/java/com/amazonaws/services/msf/SQSStreamingJob.java

```diff
@@ -0,0 +1,84 @@
+package com.amazonaws.services.msf;
+
+import com.amazonaws.services.kinesisanalytics.runtime.KinesisAnalyticsRuntime;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.flink.api.common.eventtime.WatermarkStrategy;
+import org.apache.flink.api.common.serialization.SimpleStringSchema;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.connector.source.util.ratelimit.RateLimiterStrategy;
+import org.apache.flink.connector.datagen.source.DataGeneratorSource;
+import org.apache.flink.connector.sqs.sink.SqsSink;
+import org.apache.flink.connector.sqs.sink.SqsSinkElementConverter;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.LocalStreamEnvironment;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+
+public class SQSStreamingJob {
+
+    // Name of the local JSON resource with the application properties in the same format as they are received from the Amazon Managed Service for Apache Flink runtime
+    private static final String LOCAL_APPLICATION_PROPERTIES_RESOURCE = "flink-application-properties-dev.json";
+
+    // Create ObjectMapper instance to serialise POJOs into JSONs
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    private static boolean isLocal(StreamExecutionEnvironment env) {
+        return env instanceof LocalStreamEnvironment;
+    }
+
+    /**
+     * Load application properties from Amazon Managed Service for Apache Flink runtime or from a local resource, when the environment is local
+     */
+    private static Map<String, Properties> loadApplicationProperties(StreamExecutionEnvironment env) throws IOException {
+        if (isLocal(env)) {
+            return KinesisAnalyticsRuntime.getApplicationProperties(
+                    SQSStreamingJob.class.getClassLoader()
+                            .getResource(LOCAL_APPLICATION_PROPERTIES_RESOURCE).getPath());
+        } else {
+            return KinesisAnalyticsRuntime.getApplicationProperties();
+        }
+    }
+
+    private static DataGeneratorSource<StockPrice> getStockPriceDataGeneratorSource() {
+        long recordPerSecond = 100;
+        return new DataGeneratorSource<>(
+                new StockPriceGeneratorFunction(),
+                Long.MAX_VALUE,
+                RateLimiterStrategy.perSecond(recordPerSecond),
+                TypeInformation.of(StockPrice.class));
+    }
+
+    private static SqsSink<String> createSQSSink(
```

Comment:
Can we make this a bit more general, to sink any type?
Similarly to the [Kinesis example](https://github.com/aws-samples/amazon-managed-service-for-apache-flink-examples/blob/27703164d5aec940007f598405ab1c11bc7df995/java/KinesisConnectors/src/main/java/com/amazonaws/services/msf/StreamingJob.java#L59) 

The signature can be
`<T> SqsSink<T> createSQSSink(Properties sinkProperties, SqsSinkElementConverter<T> elementConverter)` and you create the converter for your time in the main code (see my other comment below)

Nobody really send Strings and serialization is a common head scratcher for users new to Flink